features/keychain/module/keychain.js

```diff
@@ -102,7 +127,10 @@ export class Keychain {
   }
 
   async exportKey({ seedId, keyId, exportPrivate }) {
-    if (exportPrivate) this.#assertPrivateKeysUnlocked()
+    if (exportPrivate) {
```

Comment:
isn't seedId required?

Reply:
Not sure, I am not really intro'ed, but in the tests some failed without these checks, don't recall if it was this exact function, let me test that.

Reply:
we validate it a bit later: https://github.com/ExodusMovement/exodus-oss/blob/28507b6e1494deab2a0d2e5085a900fcf870214a/features/keychain/module/keychain.js#L85

i think we can remove the conditional and just assert seedId is a string

Reply:
This is a different check though, we are making sure the keys that are being exported are not locked here

Reply:
yes, but why pretend to support an undefined seedId?

Reply:
https://github.com/ExodusMovement/exodus-oss/pull/123/commits/0070939389adf63fe0aaa2e651889f5655f25f46